pkg/server/plugin/datastore/sql/sql_test.go

```diff
@@ -1551,6 +1551,8 @@ func (s *PluginSuite) TestMigration() {
 			s.Require().Len(resp.Entries, 1)
 			s.Require().Len(resp.Entries[0].DnsNames, 1)
 			s.Require().Equal("abcd.efg", resp.Entries[0].DnsNames[0])
+		case 8:
+			// todo
```

Comment:
any ideas on how to validate this? the [gorm.Dialect](https://godoc.org/github.com/jinzhu/gorm#Dialect) object has a `HasIndex()` call, but we don't have access to the underling `gorm.DB` object, since its across the plugin boundary